lms/services/course.py

```diff
@@ -357,6 +357,7 @@ def _get_copied_from_course(self, lti_params) -> Course | None:
 
         history_params = [
             "custom_Context.id.history",  # LTI 1.3
+            "custom_context_id_history",  # LTI 1.1
```

Comment:
Different naming conventions in both versions.

We don't really read this value in the real code so this has gone unnoticed.

Fixing it so we record this data when available.